config/sources/mx7dsabre.conf

```diff
@@ -0,0 +1,61 @@
+BOOTSOURCE='https://source.codeaurora.org/external/imx/uboot-imx'
+BOOTDIR='u-boot-sabre'
+BOOTBRANCH='branch:imx_v2018.03_4.14.98_2.0.0_ga'
+BOOTCONFIG="mx7dsabresd_defconfig"
+#BOOTSCRIPT='boot-sabre.cmd:boot.cmd'
+BOOTENV_FILE='sabre-default.txt'
```

Comment:
Your file is: config/bootenv/mx7dsabre-default.txt -> config/bootenv/sabre-default.txt